client/src/tests/components/UserInput.spec.js

```diff
@@ -0,0 +1,40 @@
+import expect from 'expect';
+import React from 'react';
+import { shallow } from 'enzyme';
+import { UserInput } from '../../components/common/UserInput';
+
+/**
+ * Wrapper function for the shallow function
+ *  @returns {*} JSX
+ */
+function setup() {
```

Comment:
Nitpick
You don't need this function. Simply pass in the props to the component directl